### DISCUSSION TITLE
ask for update 4701 dashboard some query expression for the lastest Metrics(like jmx_prometheus_javaagent-1.1.0.jar)

### DISCUSSION THREAD

**Discussion by GentleHe** [mweirauch/grafana-dashboards#3] [Q&A]
example: jvm_classes_loaded_classes -> jvm_classes_loaded_total

**Comment by mweirauch** [comment]
Heya. It looks to me that this metric (`jvm_classes_loaded_total`) is generated - as you have written - by a direct Prometheus Java agent. This dashboards is for visualizing metrics gathered from processes instrumented with Micrometer. You'd need to use a another dashboard for this as it would mean to mix in support for different instrumentation libraries which would be a maintenance burden.
